apps/opik-backend/src/main/java/com/comet/opik/infrastructure/LlmProviderClientConfig.java

```diff
@@ -0,0 +1,42 @@
+package com.comet.opik.infrastructure;
+
+import io.dropwizard.util.Duration;
+import io.dropwizard.validation.MinDuration;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+import java.util.concurrent.TimeUnit;
+
+@Data
+public class LlmProviderClientConfig {
+
+    public record OpenApiClientConfig(String url) {
+    }
+
+    @Min(1)
+    private Integer maxAttempts;
+
+    @Min(1)
+    private int delayMillis = 500;
+
+    @Positive private Double jitterScale;
+
+    @Positive private Double backoffExp;
+
+    @MinDuration(value = 1, unit = TimeUnit.MILLISECONDS)
```

Comment:
maybe the minimum value in milliseconds for timeouts(this and the next ones) could be higher, like at least 100? This way we protect users that dont know/understand if its seconds or milliseconds.

it can be very frustating for users setting a value like '10' imagining seconds and only getting timeouts.

Reply:
I wanted to make it as flexible as possible to not to restrict the user to any value. Also, I took some inspiration from the values in some of the core Dropwizard configs such as `DataSourceFactory`.

But your comment makes sense. I might update the values in a follow-up PR.